### PR TITLE
Format tensor on cpu

### PR DESCRIPTION
**Pull request by liujuncheng**
打印tensor需要的算子比较多，考虑到新设备可能对这些算子没有适配，所以在非CPU与CUDA的情况下，将tensor拷贝到CPU上打印

### CODE REVIEW

File: python/oneflow/framework/tensor_str.py [hunk 1]
```diff
@@ -366,7 +364,11 @@ def _gen_tensor_str_template(tensor, is_meta):
         tensor_str = "..."
         suffixes.append("size=" + str(tuple(tensor.shape)))
     else:
-        tensor_str = _tensor_str(tensor, indent)
+        if tensor.device.type != "cpu" and tensor.device.type != "cuda":
+            with flow.no_grad():
+                tensor_str = _tensor_str(tensor.to("cpu"), indent)
```
Comment (hjchen2) [inline python/oneflow/framework/tensor_str.py:369]: 最好用`tensor.detach().to("cpu")`

**Review by BBuf — APPROVED** [review]

File: python/oneflow/framework/tensor_str.py (hunk 1 above)
Comment (liujuncheng) [reply python/oneflow/framework/tensor_str.py]: > 最好用`tensor.detach().to("cpu")`

已修改

**Review by hjchen2 — APPROVED** [review]
